services/base.js

```diff
@@ -165,6 +165,17 @@ class BaseService {
     return new RegExp(fullRegex)
   }
 
+  static get _cacheLength() {
+    const cacheLengths = {
+      build: 30,
+      license: 3600,
+      version: 300,
+    }
+    return cacheLengths.hasOwnProperty(this.category)
```

Comment:
Do we actually need this extra check? Wouldn't calling `cacheLengths[this.category]` for an unsupported category return `undefined` anyway?